groups/validate.go

```diff
@@ -73,27 +91,39 @@ func TestMain(m *testing.M) {
 		}
 	}
 
-	if err := cfg.Load(*groupsPath, &rConfig); err != nil {
+	if err := groupsConfig.Load(*groupsPath, &restrictionsConfig); err != nil {
 		fmt.Printf("Could not load groups config: %v\n", err)
 		os.Exit(1)
 	}
-	os.Exit(m.Run())
+
+	// Run all the Verification Logic
+	VerifyMergedGroupsConfig(config)
+	VerifyStagingEmailLength(config)
+	VerifyDescriptionLength(config)
+	VerifyGroupConventions(config)
+	VerifyK8sInfraGroupConventions(config)
+	VerifyK8sInfraRBACGroupConventions(config)
+	VerifySecurityResponseCommitteeGroups(config)
+	VerifyNoDuplicateMembers(config)
+	VerifyHardcodedGroupsForParanoia(config)
+	VerifyGroupsWhichShouldSupportHistory(config)
 }
 
-// TestMergedGroupsConfig tests that readGroupsConfig reads all
+// VerifyMergedGroupsConfig tests that readGroupsConfig reads all
 // groups.yaml files and the merged config does not contain any duplicates.
 //
 // It tests that the config is merged by checking that the final
 // GroupsConfig contains at least one group that isn't in the
 // root groups.yaml file.
-func TestMergedGroupsConfig(t *testing.T) {
+func VerifyMergedGroupsConfig(config Config) {
 	var containsMergedConfig bool
 	found := sets.String{}
 	dups := sets.String{}
 
-	for _, g := range cfg.Groups {
+	for _, g := range groupsConfig.Groups {
 		name := g.Name
-		if name == "community" {
+
+		if name == "security" {
```

Comment:
Why did this change? Was this a data change?

Reply:
security is common for both projects right now.